sdk/storage/Azure.Storage.DataMovement.Blobs/src/AppendBlobStorageResource.cs

```diff
@@ -283,27 +283,19 @@ protected override async Task<bool> DeleteIfExistsAsync(CancellationToken cancel
             return await BlobClient.DeleteIfExistsAsync(cancellationToken: cancellationToken).ConfigureAwait(false);
         }
 
-        /// <summary>
-        /// Gets the source checkpoint data for this resource that will be written to the checkpointer.
-        /// </summary>
-        /// <returns>A <see cref="StorageResourceCheckpointData"/> containing the checkpoint information for this resource.</returns>
-        protected override StorageResourceCheckpointData GetSourceCheckpointData()
+        public override StorageResourceCheckpointData GetSourceCheckpointData()
```

Comment:
Commenting here since this is the first place this is seen in the review: I temporarily made these abstract functions `public` instead of `protected internal` because of some issues that arise from cross-compiling the checkpointer into Blobs. I will put these back in my next PR which will eliminate the cross-compilation since this PR removes the need for it (yay!).

This same thing applies to `StorageResourceCheckpointData.Serialize` which you will see later in the review. Will put that back in next PR as well.

Reply:
Seems like ci was actually ok! I think it just automatically inherits the doc from the abstract class.